### PR TITLE
Local

### PR DESCRIPTION
**Pull request by Dmambo**
# FEATURES ADDED 👍 

When the user changes the content of any input field, the data is saved to the local storage. ✔️
When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.✔️

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -171,9 +171,11 @@ <h2>Contact me</h2>
             <form action="https://formspree.io/f/xwkzweoq" method="post" class="forms" id="form">
                 <fieldset class="form-inputs">
                     <input type="text" id="name" placeholder="Your Name" maxlength="30" required />
+
                     <input type="email" id="email" placeholder="example@mail.com" required>
                     <div class="warn"></div>
                     <textarea name="message" placeholder="write your message here" maxlength="500" required></textarea>
+
                     <button type="submit" class="submit-btn">Get In Touch</button>
```
Comment (alexpuente0) [inline index.html]: - You missed the id of the `textarea`, which is causing a Null Variable error (you can see it on the browser console). This is causing that the data doesn't get stored. Please review this issue so you can move ahead.

**Review by alexpuente0 — CHANGES_REQUESTED** [review]
Hi @Dmambo and @ohyajk, 



You've done a great job so far!

In this case, however, there are still some issues that you need to work on to finish this project and advance to the next one, but you are almost there!

## Highlights
- Your code is very neatly organized

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by pinkmoon25 — APPROVED** [review]
Hi @Dmambo @ohyajk , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
professional readme :heavy_check_mark: 
Linters checks :heavy_check_mark: 
Data preserved in local storage :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — COMMENTED** [review]
## Hi @Dmambo and @ohyajk   :smiley: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Descriptive pull request message and title :heavy_check_mark: 
Linter error checks validated :heavy_check_mark: 
Commit messages are meaningful. :heavy_check_mark: 
GitHub flow observed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**